### PR TITLE
feat: add experiment registration handling

### PR DESCRIPTION
**Pull request by slawrence87544**
body: adding handling to both query from and insert into the
experiment registration table.

### CODE REVIEW

File: src/experiments.py [hunk 1]
```diff
@@ -0,0 +1,572 @@
+"""
+Copyright 2022 NOAA
+All rights reserved.
+
+Collection of methods to facilitate handling of score db requests
```
Comment (frolovsa) [inline src/experiments.py]: is this an accurate comment?

File: src/experiments.py [hunk 2]
```diff
@@ -0,0 +1,572 @@
...
+        msg = 'Invalid datetime format, must be ' \
+            f'of \'{datetime_format}\'. ' \
+            f'For example: \'{example_time}\'.' \
+            f' Error: {err}.'
+        raise ValueError(msg) from err
+    
+    return parsed_time
+
+
+def get_time_filter(filters, cls, key, constructed_filter):
```
Comment (frolovsa) [inline src/experiments.py]: add a comment on the intention of this function

File: src/experiments.py [hunk 3]
```diff
@@ -0,0 +1,572 @@
...
+        return constructed_filter
+
+    exact_match_filter = string_flt.get('exact')
+    if exact_match_filter is not None:
+        constructed_filter[key] = (getattr(cls, key) == exact_match_filter)
+
+    return constructed_filter
+
+
+def construct_filters(filters):
```
Comment (frolovsa) [inline src/experiments.py:310]: might needs some comment?

File: src/experiments.py [hunk 4]
```diff
@@ -0,0 +1,572 @@
...
+        raise TypeError(f'\'order_by\' must be a str, was {type(value)}')
+    
+    if value not in VALID_ORDER_BY:
+        raise TypeError(f'\'order_by\' must be one of {VALID_ORDER_BY}, ' \
+            f' was {value}')
+    
+    return value
+
+
+def build_column_ordering(cls, ordering):
```
Comment (frolovsa) [inline src/experiments.py:409]: add comment that this relates to SQL ORDER BY

File: tests/configs/job_example.sh [hunk 5]
```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -l
```
Comment (frolovsa) [inline tests/configs/job_example.sh]: remove this from repo. not used.

**Review by frolovsa — CHANGES_REQUESTED** [review]
see few minor comments.

**Review by frolovsa — CHANGES_REQUESTED** [review]
Can you add an example where you register an experiment from a yaml file

File: tests/configs/job_example.sh (hunk 5 above)
Comment (slawrence87544) [reply tests/configs/job_example.sh]: removed...

File: src/experiments.py (hunk 4 above)
Comment (slawrence87544) [reply src/experiments.py:409]: added method comment section

File: src/experiments.py (hunk 2 above)
Comment (slawrence87544) [reply src/experiments.py]: Added comments for get_time_filter

**Pull request merged**